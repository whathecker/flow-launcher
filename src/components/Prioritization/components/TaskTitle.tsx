import React from "react";
import { Text } from "../../Themed";
import { Typography, Color } from "../../../styles";

type TaskTitleProps = {
  title: string;
};

const TaskTitle: React.FC<TaskTitleProps> = ({ title }: TaskTitleProps) => {
  return (
    <>
      <Text style={{ ...Typography.p, fontSize: 20, color: Color.light.text }}>
        {title}
      </Text>
    </>
  );
};

export default TaskTitle;
