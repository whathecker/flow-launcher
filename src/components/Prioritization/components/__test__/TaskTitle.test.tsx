import * as React from "react";
import renderer from "react-test-renderer";
import TaskTitle from "../TaskTitle";

describe("Test TaskTitle component", () => {
  it("Should renders TaskTitle correctly", () => {
    const tree = renderer.create(<TaskTitle title={`Test title`} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
