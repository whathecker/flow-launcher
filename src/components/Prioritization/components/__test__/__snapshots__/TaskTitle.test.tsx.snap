// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test TaskTitle component Should renders TaskTitle correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#554F4F",
      },
      Object {
        "fontFamily": "Lexend-SemiBold",
        "fontSize": 18,
      },
    ]
  }
>
  Test title
</Text>
`;
